### PR TITLE
Fix missing blank space issue

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
r? @Manishearth

### CODE REVIEW

File: src/librustdoc/html/render.rs [hunk 1]
```diff
@@ -1888,11 +1888,11 @@ fn short_stability(item: &clean::Item, cx: &Context, show_reason: bool) -> Vec<S
                 if stab.unstable_reason.is_empty() {
                     stability.push(format!("<div class='stab unstable'>\
                                             <span class=microscope>🔬</span> \
-                                            This is a nightly-only experimental API. {}</div>",
```
Comment (Manishearth) [inline src/librustdoc/html/render.rs]: There's no difference between a regular space and an `&nbsp`, nbsp just lets you string together two spaces.

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by GuillaumeGomez** [comment]
@bors: r-

Some tests are failing, I'll fix them tomorrow.

**Comment by GuillaumeGomez** [comment]
Fixed.

@bors: r=Manishearth

**Pull request merged**